store/postgres/src/copy_dds.sql

```diff
@@ -0,0 +1,60 @@
+with xlat as (
```

Comment:
Yes it will have to be regenerated when the metadata schema changes. I could easily turn this into something that is dynamically generated to make that unnecessary. I wasn't sure whether it's preferrable to have to regenerate, but be able to inspect the query, or to not have to regenerate and leave the query harder to get to.